lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -480,16 +480,19 @@ def _section_course_info(course, access):
     return section_data
 
 
-def _section_membership(course, access, is_white_label):
+def _section_membership(course, access):
     """ Provide data for the corresponding dashboard section """
     course_key = course.id
     ccx_enabled = settings.FEATURES.get('CUSTOM_COURSES_EDX', False) and course.enable_ccx
+    default_enrollment_role_choices = ['Partner', 'Support', 'Learner']
+    enrollment_role_choices = configuration_helpers.get_value('MANUAL_ENROLLMENT_ROLE_CHOICES',
+                                                              default_enrollment_role_choices)
```

Comment:
please write a test to test Sites setting custom values in MANUAL_ENROLLMENT_ROLE_CHOICES and make sure they get used if set